lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -377,6 +343,13 @@ struct TypeLoweringVisitor : public FIRRTLVisitor<TypeLoweringVisitor, bool> {
 
   /// The builder is set and maintained in the main loop.
   ImplicitLocOpBuilder *builder;
+
+  // After a bundle type is lowered to its field elements, they can have a
+  // reference to a NonLocalAnchor. This new lowered field element will get a
+  // new symbol, and the corresponding NLA needs to be updatd with this symbol.
+  // This is a list of the NLA name  which needs to be updated, to the new
+  // lowered field symbol name.
+  SmallVector<std::pair<StringAttr, StringAttr>> &nlaNameToNewSymList;
```

Comment:
Nit: This `std::pair<StringAttr, StringAttr>` gets used a lot. Maybe a `struct` in an anonymous namespace would be cleaner (and slightly clarify the use of `.first` and `.second` at the end of the pass.

Reply:
Moved it to a struct type.